src/components/OutlinedButton.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <a
+    :disabled="disabled"
+    class="button-link"
+    :href="href"
+    :target="target"
+    :title="title"
+    @click="$emit('click', $event)"
+  >
+    <template>
+      <slot />
+    </template>
+  </a>
+</template>
+
+<script>
+
+export default {
+  props: {
+    disabled: Boolean,
+    href: { type: String, default: '' },
+    target: { type: String, default: '_self' },
+    title: { type: String, default: '' },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.button-link {
+  background: transparent;
```

Comment:
Do links have some background by default? 🤔

Reply:
I don't think so. This change was removed.